core/src/main/java/org/apache/iceberg/avro/AvroSchemaUtil.java

```diff
@@ -90,6 +90,25 @@ static boolean hasIds(Schema schema) {
     return AvroCustomOrderSchemaVisitor.visit(schema, new HasIds());
   }
 
+  /**
+   * Check if any of the nodes in a given avro schema is missing an ID recognizable by Iceberg core API
+   *
```

Comment:
To separate paragraphs in Javadoc, add `<p>` to this line. No need for closing tags.

Reply:
Added